README.md

```diff
@@ -38,39 +38,28 @@
 
 ### Plugin Marketplace
 
-1. Head to the [Plugin Marketplace](https://plugins.jetbrains.com/) (<kbd>Settings</kbd> → <kbd>Plugins</kbd> → <kbd>
-   Marketplace</kbd>)
-2. Search for **"Catppuccin Icons"**
-3. Install Plugin
+1. Install the [Catppuccin Icons plugin](https://plugins.jetbrains.com/plugin/23029-catppuccin-icons) from the JetBrains Marketplace.
```

Comment:
I highly doubt many people navigate to the webpage to actually install the plugin. I'd like to keep the steps for searching within the IDE.

Reply:
The previous instructions mentioned going to the plugin marketplace and gave a link though anyway? Also can't you click the link and open the installation page in the editor? Or is VSCode just that much better?

Reply:
The link is just there to make sure people have an easy way of looking at the plugin on the marketplace. 

JetBrains IDE's don't run on electron and doesn't just open the installation page in the editor, you need to navigate through some settings for it. There is a button on the webpage that allows you to install to your IDE but I've found that it's quite buggy sometimes and don't imagine many people click that. 

I'd keep the steps in the README :+1: